src/jazkarta/easyformplugin/savecontent/templates/listing.pt

```diff
@@ -0,0 +1,153 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en"
+    xmlns:tal="http://xml.zope.org/namespaces/tal"
+    xmlns:metal="http://xml.zope.org/namespaces/metal"
+    xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+    lang="en"
+    metal:use-macro="context/main_template/macros/master"
+    i18n:domain="plone">
+
+<metal:block fill-slot="content-title">
+  <h1 class="documentFirstHeading">
+    <span tal:content="context/Title">Title</span><br>
+    <a class="button small" href="${context/aq_parent/absolute_url}">Back to form<a>
```

Comment:
I think we can just use the listing macro here and fill this content-title slot to avoid duplicating all this code.

Reply:
@alecpm I removed everything from the template save the macro and the title slot